### DISCUSSION TITLE
Increase voting max

### DISCUSSION THREAD

**Discussion by ackernaut** [ParabolInc/parabol#6360] [Feedback]
### TL;DR
- Currently we have an upper limit of 12 votes
- Folks must click to set the total number of votes, and max per issue
![Screen Shot 2022-04-06 at 13 47 29](https://user-images.githubusercontent.com/307286/162046806-598fa327-fc79-4afe-9cb4-e10cf8a03c5f.png)
![Screen Shot 2022-04-06 at 13 47 35](https://user-images.githubusercontent.com/307286/162046809-48119127-7f61-41c5-8281-b13d0e53e4fb.png)
![Screen Shot 2022-04-06 at 13 47 40](https://user-images.githubusercontent.com/307286/162046811-c1ee6899-5fb9-4324-b540-61d02421c715.png)

- Folks could use more votes depending on their voting framework
- We could explore different UIs that support dialing in an larger number more quickly
- How might we make better suggested voting constraints that match the number of participants and/or comments or groups?

[See this thread](https://parabol.slack.com/archives/C836NA350/p1649128691532659)

>is it possible to have increase the maximum votes in a retro beyond 12? Some folks were trying to do MOSCOW modelling using Parabol.

>…around 100 would give him the flexibility he was looking for (so 99ish would likely work).  He mentioned that potentially having a skip 10 button or similar might fix having to click the button over and over again.  You could also make it a text box but that has input validation issues you might not want to deal with.
